rllib/algorithms/bandit/tests/test_bandits.py

```diff
@@ -64,6 +72,85 @@ def test_bandit_lin_ucb_compilation(self):
                 self.assertTrue(results["episode_reward_mean"] == 10.0)
                 trainer.stop()
 
+    def test_bandit_convergence(self):
+        TRAINING_ITERATIONS = 50
+        ARM_COUNT = 2
+        BETTER_ARM_PROBABILITY = 0.3
+        ACTION_TO_REWARD_PROBABILITY = {0: 0.1, 1: BETTER_ARM_PROBABILITY}
+
+        class NonContextualBanditEnv(gym.Env):
```

Comment:
Cleaned this up a little in the meantime on our end

```
ARM_COUNT = 2
GOOD_ARM_REWARD_PROBABILITY = 0.3
DUMMY_OBSERVATION = 0
REWARD = 1.0

REWARD_PROBABILITY_FOR_ACTION = {
    0: 0.1,                             # bad arm
    1: GOOD_ARM_REWARD_PROBABILITY      # good arm
}


class NonContextualBanditEnv(gym.Env):
    def __init__(self, config: EnvContext):
        self.action_space = Discrete(ARM_COUNT) # bad_arm / good_arm

        # TODO actually no features - can this be removed?
        self.observation_space = Discrete(1)

    def reset(self):
        return DUMMY_OBSERVATION

    def step(self, action):
        reward = np.random.binomial(REWARD, REWARD_PROBABILITY_FOR_ACTION[action])
        return DUMMY_OBSERVATION, reward, True, {"regret": GOOD_ARM_REWARD_PROBABILITY - REWARD_PROBABILITY_FOR_ACTION[action]}
```

and in a different script I use a more flexible implementation for a configurable number of arms
```
DUMMY_OBSERVATION = 0
REWARD = 1.0

class NonContextualBanditEnv(gym.Env):
    def __init__(self, config: EnvContext):
        self.reward_probability_for_action = config['reward_probability_for_action']
        self.highest_reward_probability = max(self.reward_probability_for_action.values())

        self.action_space = Discrete(len(self.reward_probability_for_action)) 

        # TODO actually no features - can this be removed?
        self.observation_space = Discrete(1)

    def reset(self):
        return DUMMY_OBSERVATION

    def step(self, action):
        reward = np.random.binomial(REWARD, self.reward_probability_for_action[action])
        return DUMMY_OBSERVATION, reward, True, {"regret": self.highest_reward_probability - self.reward_probability_for_action[action]}
```